docs/modules/reference/pages/configuration/file-editor.adoc

```diff
@@ -1,66 +1,68 @@
-[[file-editor]]
-= Manage Configuration Files in the UI
-
-The File Editor, available in the new UI, lets users view and edit configuration files (`.xml`, `.cfg`, `.drl`, etc.) with the OpenNMS UI instead of having to edit them directly in the file system (for example in the `/etc` directory).
 
-NOTE: Making changes to configuration files will affect how OpenNMS operates and incorrectly editing a file could result in features no longer working or not working as intended.
+[[file-editor]]
+= File Editor
 
-To access the File Editor, go to the new UI, for example by clicking on "UI Preview", then navigating to "File Editor".
+The File Editor page, available in the new web UI, lets you view and edit configuration files (`.xml`, `.cfg`, `.drl`, and so on) in the {page-component-title} UI instead of having to edit them in your local file system (for example, by locating the file in your `/etc` directory and editing it with a text editor).
```

Comment:
```suggestion
The File Editor page lets you view and edit configuration files (`.xml`, `.cfg`, `.drl`, and so on) in the {page-component-title} UI instead of having to edit them in your local file system (for example, by locating the file in your `/etc` directory and editing it with a text editor).
```